CHANGELOG.rst

```diff
@@ -2,6 +2,11 @@
 Changelog for package power_msgs
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+Forthcoming
```

Comment:
0.3.1 we'll tag it afterwards

Reply:
this breaks API doesn't it?
So it should probably be a 0.4.0 or 1.0.0

I'll use `catkin_prepare_release --bump` to do all this for us though.